Automattic/Gravatar-SDK-iOS
# Updating to latest openapi spec

Closes #

### Description

Updating to latest openapi spec

![CleanShot 2024-10-04 at 16 49 22@2x](https://github.com/user-attachments/assets/c45114c5-a38c-41bf-ab0c-baa977a2a0ab)


### Testing Steps

- Be sure to have the Gravatar API key set up on the project.
- Run the UIKit demo app.
- Go to `Fetch Profile` demo screen.
- Fetch the profile which has a verified account linked.
  - Check that the verified account information appears on the info text.
